Frontend/ui-common/src/Components/RideCompletedCard/View.purs

```diff
@@ -267,7 +268,7 @@ customerIssueView config push =
         , gravity CENTER
         , margin $ MarginTop 15
         , orientation VERTICAL
-        , visibility if config.customerIssueCard.selectedYesNoButton == 0 then VISIBLE else GONE
+        , visibility $ boolToVisibility $ config.customerIssueCard.selectedYesNoButton == if config.customerIssueCard.isNightRide then 1 else 0
```

Comment:
@PraveenGongada can we use `boolToInt` here also. for `if config.customerIssueCard.isNightRide then 1 else 0`